pkg/volume/volume_windows.go

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Does this file need some flag to only build for windows?

Reply:
I was told no flag needed for windows.

Reply:
I find CI does run windows build, see

```
I0826 07:47:28.050] +++ [0826 07:43:15] windows/amd64: go build started
I0826 07:47:28.050] +++ [0826 07:47:28] windows/amd64: go build finished
I0826 07:47:28.059] Makefile:92: recipe for target 'all' failed
I0826 07:47:28.062] Makefile:419: recipe for target 'cross' failed
```